DependencyInjection/Compiler/SystemInfoCollectorPass.php

```diff
@@ -37,4 +44,22 @@ public function process(ContainerBuilder $container)
         $infoCollectorRegistryDef = $container->findDefinition('support_tools.system_info.collector_registry');
         $infoCollectorRegistryDef->setArguments([$infoCollectors]);
     }
+
+    private function processSystemInfo(ContainerBuilder $container)
+    {
+        if (!$container->getParameter('support_tools.promote_platform.enabled')) {
```

Comment:
Partially disregard my previous comment, I missed that.
Still, could it be exposed as a semantic config (probably mapped to this parameter later on)?
Could we keep consistent naming, starting with `ezplatform.` prefix?

Reply:
> Could we keep consistent naming, starting with ezplatform. prefix?

let's maybe wait a few months with such efforts :)


For the other remarks, I'll come back to those once the feature itself has been discussed.

Reply:
Back at this, seems @SylvainGuittard is good with doing this via optional header.

> Still, could it be exposed as a semantic config (probably mapped to this parameter later on)?

You mean plan symfony semantic config right? yeah we can do that, easier to debug.
SA config however I don't see a need for here

> Could we keep consistent naming, starting with ezplatform. prefix?

So what would you recommend as 2.5 name here for the semantic config? _(master name we can discuss on separate channels once you have moved forward on current NS / naming proposal maybe)_